tokens/governance/treasury-spend.md

```diff
@@ -0,0 +1,93 @@
+
+![Proposal Roadmap](/images/tokens/governance/treasury-proposals/treasury-proposal-roadmap.png)
+
+## Submitting your Idea to the Forum {: #submitting-your-idea-to-the-forum }
+
+It's highly recommended that you preface any proposal with a post on [Moonbeam's Community Forum](https://forum.moonbeam.foundation/){target=_blank}. This especially important for treasury proposals because of the irrevocable bond required to submit a treasury spend proposal. You can submit your idea for a treasury spend proposal by following [these instructions](https://moonbeam.network/blog/using-moonbeam-community-forum/){target=_blank}. 
+
+![Moonbeam Forum Home](/images/tokens/governance/treasury-proposals/treasury-proposal-1.png)
+
+## Proposing an Action {: #proposing-an-action } 
+
+This guide focuses on the mechanics of submitting an on-chain treasury proposal after you have worked with the Treasury Council to refine your idea. If you haven't yet completed the prior steps of the treasury proposal process, please take a moment to review the [guidelines of the community treasury program](https://github.com/moonbeam-foundation/treasury/blob/main/interim/interim_treasury_proposal.md){target=_blank} and evaluate your standing. Engaging the community and revising your proposal based on feedback received is the most critical piece of your proposal. Submitting the on-chain treasury proposal is the easy part, demonstrated in the following guide. 
+
+To get started, head to the **Governance** section and click on **Treasury**. Then click **Submit Proposal**.
+
+![Treasury home](/images/tokens/governance/treasury-proposals/treasury-proposal-2.png)
+
+Then, take the following steps:
+
+1. Select the account that will be the creator of the treasury proposal. Make sure that the account selected here is sufficiently funded with your proposal bond
+2. Select the beneficiary of the proposal. This should correspond to the account indicated in your forum post.
+3. Indicate the value of the spend proposal. The required bond is {{ networks.moonbeam.treasury.proposal_bond}}% of the proposal's value. The bond committed to a treasury spend proposal is irrevocable. 
+4. Review each field for accuracy then press **Submit proposal** and sign the transaction
+
+![Submitting a treasury spend proposal](/images/tokens/governance/treasury-proposals/treasury-proposal-3.png)
+
+After the submitting your treasury proposal, you can refresh Polkadot JS Apps and you should see your proposal listed. If you don't see your proposal appear here, your account may not have contained sufficient funds for the required bond. 
+
+Optionally, you can login now to [Polkassembly](https://moonbeam.polkassembly.network/){target=_blank} with the same account that you used to create the treasury spend proposal. Polkassembly auto-generates a post for each proposal, however, you'll likely want to edit the title and description to provide more context for community members. 
```

Comment:
You may want to add the following bit:

Then on Polkassembly, as the proposer, you can edit the description of the proposal to include a link to the proposal discussion on the Moonbeam Discussion Forum.

And this:

After submitting the on-chain proposal, you’ll need to edit your proposal on the Moonbeam Discussion Forum. 

Grabbed both of these sentences from this page: https://moonbeam.network/blog/using-moonbeam-community-forum/

Reply:
Basically they just need to update the forum to submitted and whatnot